admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+===== Pod Affinity
+
+The following example demonstrates pod affinity for pods with matching labels and label selectors.
+
+* The pod *team4* has the label `team:4`.
++
+----
+$ cat team4.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: team4
+  labels:
+     team: "4"
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The pod *team4a* has the label selector `team:4` under `podAffinity`.
++
+----
+$ cat pod-team4a.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: team4a
+spec:
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: team
+            operator: In
+            values:
+            - "4"
+        topologyKey: kubernetes.io/hostname
+  containers:
+  - name: pod-affinity
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The *team4a* pod is scheduled on the same node as the *team4* pod.
+
+
+[[admin-guide-sched-affinity-examples2]]
+==== Pod Anti-Affinity
+
+The following example demonstrates pod anti-affinity for pods with matching labels and label selectors.
+
+* The pod *pod-s1* has the label `security:s1`.
++
+----
+cat pod-s1.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s1
+  labels:
+    security: s1
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The pod *pod-s2* has the label selector `s1` under `podAntiAffinity`.
++
+----
+cat pod-pod-s2.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-s2
+spec:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+            matchExpressions:
+            - key: security
+              operator: In
+              values:
+              - s1
+          topologyKey: kubernetes.io/hostname
+  containers:
+  - name: pod-antiaffinity
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The pod *pod-s2* cannot be scheduled on the same node as *pod-s1*.
+
+[[admin-guide-sched-affinity-examples3]]
+==== Pod Affinity with no Match
+
+The following example demonstrates pod affinity for pods without matching labels and label selectors.
+
+
+* The pod *pod-s1* has the label `security:s1`.
++
+----
+$ cat pod-s1.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-s1
+  labels:
+    security: s1
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The pod *pod-s2* has the label selector `security:s2`.
++
+----
+$ cat pod-s2.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-s2
+spec:
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: security
+            operator: In
+            values:
+            - s2
+        topologyKey: kubernetes.io/hostname
+  containers:
+  - name: pod-affinity
+    image: docker.io/ocpqe/hello-pod
+----
+
+* The pod *pod-s2* cannot be scheduled on the same node as `pod-s1`.
```

Comment:
"The pod *pod-s2* will not be scheduled unless there is a node with a pod that has a label with security=s2."